test/e2e/storage/persistent_volumes-local.go

```diff
@@ -201,12 +202,21 @@ var _ = utils.SIGDescribe("PersistentVolumes-local ", func() {
 				}
 				setupStorageClass(ctx, config, &testMode)
 				testVols := setupLocalVolumesPVCsPVs(ctx, config, testVolType, config.randomNode, 1, testMode)
-				testVol = testVols[0]
+				if len(testVols) > 0 {
+					testVol = testVols[0]
+				} else {
+					ginkgo.By("setupLocalVolumesPVCsPVs fail")
+					os.Exit(0)
```

Comment:
Maybe we could call the `framework.Failf` function instead of `os.exit` function directly?
/cc @pohly

Reply:
You are right. Fail the test, but don't kill the entire test suite run.

Reply:
Gotcha, i correct the os.exit.